src/OrbitGl/LineGraphTrack.h

```diff
@@ -17,13 +17,7 @@ namespace orbit_gl {
 template <size_t Dimension>
 class LineGraphTrack : public GraphTrack<Dimension> {
  public:
-  explicit LineGraphTrack(CaptureViewElement* parent, TimeGraph* time_graph,
```

Comment:
Happy for this (and a bit surprised)

Reply:
Yep, I learned this trick from @beckerhe :)